spikes/aks-secure-baseline/README.md

```diff
@@ -138,3 +141,24 @@ az acr import --source ghcr.io/retaildevcrews/ngsa-app:beta -n $ACR_NAME
 az acr import --source ghcr.io/retaildevcrews/ngsa-lr:beta -n $ACR_NAME
 
 ```
+
+Fluentbit setup
+
+```bash
+
+# create fluentbit namespace
+kubectl create namespace fluentbit
+
+Ngsa_Log_Analytics_RG="rg-enterprise-networking-hubs"
+Ngsa_Log_Analytics_Name="la-hub-eastus2-lq7hlzxsovd4c"
```

Comment:
nit: I would advise against storing these values, how about using placeholders (i.e., <<your-rg-name>>, <<log-analytics-instance-name>>) and leave it to whoever wants to use this script to plug their values in?